worlds/witness/static_logic.py

```diff
@@ -204,15 +201,15 @@ def parse_items(self):
 
     @lazy
```

Comment:
I probably just had a moment of really missing lazy properties from Kotlin and was like "Hey I wonder if you can hack this into Python" lol

Reply:
I don't really think it's that bad, though, so I think I'm gonna leave that consideration outside of the scope of this PR

Reply:
I feel like sigma_export, sigma_normal and vanilla should be `@staticmehtod`s?

Reply:
It could be possible to make them static, I think. Is there some benefit to that? An object of StaticWitnessLogic is created in the WitnessLogic class as a class variable (not an instance variable). The entire StaticWitnessLogic class is basically treated as a singleton. 

In fact, with my current implementation of it, I could probably get rid of the StaticWitnessLogic class... It has literally only static members, so it should just be a submodule probably.

Reply:
I think I'd probably want to do that as a seperate PR though, because that's probably going to affect multiple classes